src/test/kotlin/com/terraformation/backend/customer/db/FacilityStoreTest.kt

```diff
@@ -84,7 +84,7 @@ internal class FacilityStoreTest : DatabaseTest(), RunsAsUser {
     every { user.canUpdateSubLocation(any()) } returns true
     every { user.canUpdateTimeseries(any()) } returns true
 
-    timeZone = insertTimeZone()
+    timeZone = ZoneId.of("${ZoneId.of("Pacific/Honolulu")}")
```

Comment:
This seems a little ugly but is harmless for now; we can clean it up later.

Reply:
Easy fix